src/IotHub/IotHub/IotHub/GetAzureRmIotHubKey.cs

```diff
@@ -39,20 +52,26 @@ public class GetAzureRmIotHubKey : IotHubBaseCmdlet
         [Parameter(
             Position = 1,
             Mandatory = true,
+            ParameterSetName = ResourceParameterSet,
             ValueFromPipelineByPropertyName = true,
             HelpMessage = "Name of the Iot Hub")]
         [ValidateNotNullOrEmpty]
         public string Name { get; set; }
 
         [Parameter(
-            Position = 2,
```

Comment:
@anusapan removing this `Position` is the correct thing to do when it comes to PowerShell patterns, but removing it now will be a breaking change, so I'd advise reverting this change.

Reply:
Agreed... reverted